File: Ejercicios de Semana Santa/ejercicios de semana santa java/Four java/Library/LicenciaA.java
package Library;

public class LicenciaA extends Licencia {
    public LicenciaA() {
        this.tipo = "A";
        this.costo = 800000;
        this.duracion = 6;
    }

    public void mostrarDetalles() {
        System.out.println("\n--- Detalles de la Licencia ---");
        System.out.println("Tipo: " + tipo + " (Motocicletas)");
        System.out.println("Costo: " + String.format("%,.2f COP", costo));
        System.out.println("Duración: " + duracion + " meses");
    }
}

File: Ejercicios de Semana Santa/ejercicios de semana santa php/Four php/Library/LicenciaA.php
<?php
require_once 'Licencia.php';

class LicenciaA extends Licencia {
    public function __construct() {
        $this->costo = 900000;
        $this->duracion = 6;
    }

    public function mostrarServicios() {
        echo "Licencia A: motocicletas.\n";
    }
}
?>